contents/node/nodejs.md

```diff
@@ -0,0 +1,203 @@
+
+There are other useful properties in `package.json` as described below.
+
+### dependencies and devDependencies
+
+The `dependencies` property specifies dependencies needed in production while the `devDependencies` property specifies dependencies needed in development.
+
+```json
+{
+  "dependencies": {
+    "express": "~3.0.1",
+    "sequelize":"latest",
+    "bluebird": "^3.4.7",
+    "angular":"latest",
+    "angular-ui-router": "~0.2.11",
+    "path":"latest",
+  },
+  "devDependencies": {
+    "eslint": "^4.16.0",
+    "eslint-config-airbnb-base": "^12.1.0",
+    "eslint-plugin-import": "^2.8.0"
+  }
+}
+```
+The `dependencies` property maps to an object that has the name and version range for each dependency. It is common to find carets (`^`) and tildes (`~`) in the version range. To understand what they are for, checkout [NodeSource's article on semvers](https://nodesource.com/blog/semver-a-primer/). It is important to specify an appropriate version range to ensure consistency and so that users and developers will have compatible versions. For example, it is not advisable to always use the latest version of a dependency as it might introduce breaking changes (e.g. deprecate API) and the use of different versions by different developers and users might make debugging harder.
+
+It is also possible (and easier) to install a new dependency and update `package.json` directly from the command line with
+```
+$ npm install <package_name> --save
+```
+When a dependency is installed, the package's code will be added to the local `/node_modules` folder.
+
+You can find a wide range of useful packages on [npmjs](https://www.npmjs.com/). The [module section](#node-module-system) describes how to import packages/modules in your code.
+
+### scripts
+
+The `scripts` property is a useful tool that can be used to perform repetitive tasks such as testing and building. It takes in an object. For each property, the key is the given name of the command and the corresponding value is the actual command that will be run.
+
+For example, inside your `package.json`, you might have
+```json
+{
+  "scripts": {
+      "build": "node app.js",
+      "lint": "eslint **/*.js",
+      "lint-fix": "eslint --fix **/*.js"
+  }
+}
+```
+Running `npm run build` in the command shell will execute `node app.js` and similarly `npm run lint-fix` will fix your linting errors in your JavaScript files if you have eslint installed. npm scripts are much more powerful - you can use them as shortcuts to [uglify JavaScript, auto-prefix CSS](https://css-tricks.com/why-npm-scripts/) and even execute multiple tasks at a go. To find out more about how you can use npm scripts, refer to [freeCodeCamp's Introduction to NPM Scripts](https://medium.freecodecamp.org/introduction-to-npm-scripts-1dbb2ae01633).
+
+### Others
+
+For more information about `package.json`, refer to [npm official documentation](https://docs.npmjs.com/files/package.json).
```

Comment:
The section on NPM still feels like you are trying to teach how to use it instead of explaining how useful it is.